packages/rest/tsup.config.ts

```diff
@@ -2,6 +2,6 @@ import { esbuildPluginVersionInjector } from 'esbuild-plugin-version-injector';
 import { createTsupConfig } from '../../tsup.config.js';
 
 export default createTsupConfig({
-	entry: ['src/index.ts', 'src/strategies/*.ts'],
+	entry: ['src/agnostic.ts', 'src/web.ts', 'src/strategies/*.ts'],
```

Comment:
Why the rename to agnostic? This can just stay as `index` no? The default therefore is agnostic.

Reply:
Can rename it back to index, sure